kin-devplatform-sdk/src/main/java/kin/devplatform/network/api/RestorableWalletApi.java

```diff
@@ -0,0 +1,54 @@
+package kin.devplatform.network.api;
+
+import static kin.devplatform.core.network.ApiClient.GET;
+
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.Map;
+import kin.devplatform.ConfigurationImpl;
+import kin.devplatform.core.network.ApiCallback;
+import kin.devplatform.core.network.ApiClient;
+import kin.devplatform.core.network.ApiException;
+import kin.devplatform.network.model.RestorableWalletRequest;
+import okhttp3.Call;
+
+public class RestorableWalletApi {
```

Comment:
I think it should be added to the existing AuthApi (where update wallet already exists)

Reply:
No problem.
Fixed.